src/proof/proof_rule.h

```diff
@@ -228,7 +228,7 @@ enum class PfRule : uint32_t
   // The remaining rules in this section have the signature of a "trusted rule":
   //
   // Children: ?
-  // Arguments: (F)
+  // Arguments: (F, ...)
```

Comment:
I think we should mention for `THEORY_PREPROCESS_LEMMA` that there is another argument, and what it means.

Reply:
Do the remaining lemmas use the additional arguments? If not, remove.